codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ServerDowngraderTest.java

```diff
@@ -310,8 +348,8 @@ public void testPassThroughOther() throws Exception {
         assertTrue(ch.writeInbound(reset));
         assertTrue(ch.writeInbound(goaway));
 
-        assertEquals(ch.readInbound(), reset);
-        assertEquals(ch.readInbound(), goaway);
+        assertEquals(reset, ch.readInbound());
+        assertEquals(goaway, ch.readInbound());
```

Comment:
also comparing to `goaway` may be risky because it should be released when it was written above.